fhir-server/src/test/java/com/ibm/fhir/server/test/CapabilitiesTest.java

```diff
@@ -83,12 +85,16 @@ void testBuildCapabilityStatement_resources_filtered() throws Exception {
         assertEquals(capabilityStatement.getRest().size(), 1, "Number of REST Elements");
         CapabilityStatement.Rest restDefinition = capabilityStatement.getRest().get(0);
 
-        assertRestDefinition(restDefinition, 2, 2, 1, 0, 4, 0, 1);
+        assertRestDefinition(restDefinition, 4, 2, 2, 1, 1, 5, 1, 1);
```

Comment:
sometimes I really wish you could name parameters in a method call like you can with Ada!